test/docfx.Test/lib/MaskUtilityTest.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace Microsoft.Docs.Build
+{
+    public static class MaskUtilityTest
+    {
+        [Fact]
+        public static void HideSecret()
+        {
+            var githubToken = TestUtility.RandomSHA1Hash();
```

Comment:
why this `RandomSHA1Hash` method? Can `Guid.NewGuid().ToString()` work?

Reply:
I checked that github PAT is a string of 40 chars with [0-9a-f], more like a SHA1 hash